packages/perseus/src/widgets/matcher/matcher.tsx

```diff
@@ -157,18 +159,13 @@ export class Matcher extends React.Component<Props, State> implements Widget {
             );
         }
 
-        // Use the same random() function to shuffle both columns sequentially
-        const rng = seededRNG(this.props.problemNum as number);
-
-        let left;
-        if (!this.props.orderMatters) {
-            // If the order doesn't matter, don't shuffle the left column
-            left = this.props.left;
-        } else {
-            left = shuffle(this.props.left, rng, /* ensurePermuted */ true);
-        }
-
-        const right = shuffle(this.props.right, rng, /* ensurePermuted */ true);
+        /* TODO(LEMS-2841):
+            Once the getRadioPublicWidgetOptions function gets connected to the
```

Comment:
`getRadioPublicWidgetOptions`? Not `getMatcherPublicWidgetOptions`?

Reply:
Eeek, maybe a copy paste error? Did I even work on radio? lol. 

Updated. Thanks! :D